Cargo.toml

```diff
@@ -117,6 +117,9 @@ console = [
     "dep:console-subscriber",
     "tokio/tracing",
 ]
+docker = [
+    "default",
```

Comment:
This should be `docker = []` and the `default` feature should contain `"docker"`.

Reply:
Done